### PR TITLE
dl orbits from asf first

### PR DESCRIPTION
**Pull request by bbuzz31**
download orbits from asf first then fall back to esa

### CODE REVIEW

File: tools/RAiDER/s1_orbits.py [hunk 1]
```diff
@@ -58,7 +58,19 @@ def get_orbits_from_slc_ids(slc_ids: List[str], directory=Path.cwd()) -> List[Pa
...
+def download_eofs(dts:list, missions:list, save_dir:str):
+    """ Wrapper to first try downloading from ASF """
+    import eof.download
+    try:
+        orb_files = eof.download.download_eofs(dts, missions, save_dir=save_dir, force_asf=True)
+    except:
+        logger.warning('Could not download orbit from ASF, trying ESA...')
+        orb_files = eof.download.download_eofs(dts, missions, save_dir=save_dir, force_asf=False)
+
+    return orb_files
```
Comment (dbekaert) [inline tools/RAiDER/s1_orbits.py:76]: @bbuzz31 is this a list of dates or single date. We might want to try and pull whatever we can pull from ASF and then what fails from ESA?

**Comment by bbuzz31** [comment]
@dbekaert we're now pulling each orbit individually, first checking if it's ASF then falling back to ESA. It's a bit ugly because we can't leverage the simultaneous jobs as `sentineleof` has built in, but ESA limits to 4 anyway. I tried checking against existing orbit files so that we could keep this functionality, but it is too complicated to figure out the different combinations that we currently support

**Comment by bbuzz31** [comment]
@cmarshak I had to change the orbit reader test to support the new multiple calls. it works but I have no real idea what I'm doing with this mocker stuff. let me know if it should be changed

**Comment by dbekaert** [comment]
Once we have the fix on the sentinel reader for the coockie we can just
revert back to our original approach which would be indeed cleanest.

**Review by jlmaurer — COMMENTED** [review]
LGTM but when I pulled and tried to run locally I'm getting `ValueError: get_sv: I cannot parse the statevector file ["[PosixPath('/Users/jlmd9g/Documents/raider_test/aria_products/orbits/S1A_OPER_AUX_POEORB_OPOD_20220213T081624_V20220123T225942_20220125T005942.EOF')]"]`. Any ideas?

**Comment by cmarshak** [comment]
> @cmarshak I had to change the orbit reader test to support the new multiple calls. it works but I have no real idea what I'm doing with this mocker stuff. let me know if it should be changed

The idea is to slightly increase the runtime of tests - have the data staged and mock/inject into the workflow. Also can make sure you are calling certain functions and not calling others e.g. if you want to make sure you never enter a particular part of a workflow because say HRRR is not available, you can do that. You can let me know if you have specific questions.

**Comment by cmarshak** [comment]
> @cmarshak I had to change the orbit reader test to support the new multiple calls. it works but I have no real idea what I'm doing with this mocker stuff. let me know if it should be changed

The idea is to slightly increase the runtime of tests - have the data staged and mock/inject into the workflow. Also can make sure you are calling certain functions and not calling others e.g. if you want to make sure you never enter a particular part of a workflow because say HRRR is not available, you can do that. You can let me know if you have specific questions.

**Comment by cmarshak** [comment]
Resolves #634.

**Review by jlmaurer — APPROVED** [review]
Looks to be passing now!

**Pull request merged**